### ISSUE TITLE
1.4.11 hover state clarification

### ISSUE THREAD

**Issue by mfairchild365** [w3c/wcag#849]
I put together [several examples](https://skitter-wavelength.glitch.me/) to help me understand and practice 1.4.11 as it applies to the hover state. This raised a couple of questions.

1. Does scenario 3 pass or fail? The button's background loses its contrast with the adjacent color on hover, but there is not another button or interactive component adjacent to it. If there was another button adjacent to it, would it fail 1.4.11?
2. Does Scenario 6 pass or fail? In this case, the background for the button is removed entirely on hover, which means that it loses its contrast with the adjacent color.

I'm also happy to receive confirmation/corrections on the rest of the scenarios.

**Comment by awkawk** [comment]
I don't think that any of these fail.

**Comment by LaurenceRLewis** [comment]
IMO FWIW I find, 'Scenario 1: default button with no hover styles', where there is no hover state and no cursor change to indicate the button is actionable problematic. If there is no textual description / name to describe the purpose,  or that it is a functioning button and not a decoration element, how will the user perceive this? 

I find similar with, 'Scenario 6: hover state results in loss of background/border differentiation between component and the rest of the background', the underline contrast against the button background should meet the 3.1:1 contrast ratio as the button background is adjacent to the underline.

Both web page background and the default cursor for the `<button>` element are the same; and therefore there is no visible change on hover.

**Comment by awkawk** [comment]
@LaurenceRLewis I don't disagree, but these are problems that would affect all users and aren't specific to any disability. I don't think that anyone in the WG would _recommend_ all of these approaches, but the success criterion language doesn't forbid  it either.

It is also possible to have buttons that are transparent (e.g. the main nav on https://www.funka.com/) so the user makes a determination that the items are clickable without the quiet or hover state having sufficient contrast. The text in the button does meet 4.5:1, but not the button background.

**Comment by alastc** [comment]
The group discussed this today and agree this response:
<hr>
The problems outlined in the examples are problems that would affect all users and aren't specific to any disability. We would not recommend all of these approaches, but the success criterion language doesn't forbid it either. These do not fail the success criteria.
